internal/olm/operator/registry/operator_installer.go

```diff
@@ -56,19 +56,13 @@ func (o OperatorInstaller) InstallOperator(ctx context.Context) (*v1alpha1.Clust
 	}
 
 	// TODO: OLM doesn't appear to propagate the "READY" connection status to the catalogsource in a timely manner
-	// even though its catalog-operator reports a connection almost immediately. This condition either needs to
-	// be propagated more quickly by OLM or we need to find a different resource to probe for readiness.
-	//
+	// even though its catalog-operator reports a connection almost immediately. This condition
+	// either needs to be propagated more quickly by OLM or we need to find a different resource to probe for readiness.
+	// wait for catalog source to be ready
 	// if err := o.waitForCatalogSource(ctx, cs); err != nil {
 	// 	return nil, err
 	// }
 
-	log.Infof("OperatorInstaller.CatalogSourceName: %q\n", o.CatalogSourceName)
```

Comment:
Are these logs being removed for a good reason? I thought we wanted to provide as much information as possible to the user.

Reply:
I included those logs in the initial boilerplate PR so I could run the CLI and see that everything was propagating through correctly.

I told @rashmigottipati they could be removed now that we have the real implementations in place. I'm not opposed to leaving informational logging like this in, but I never intended for _these_ specific logs to stick around.

Reply:
I've added some informational logs. Overall, it looks like below: 

> INFO[0008] Successfully created registry pod: quay-io-joelanford-example-operator-bundle-0-1-0
INFO[0008] Created CatalogSource: example-operator-catalog
INFO[0008] Created OperatorGroup: operator-sdk-og
INFO[0008] Created Subscription: example-operator-v0-1-0-sub
INFO[0012] Approved InstallPlan install-47hnc for the Subscription: example-operator-v0-1-0-sub
INFO[0012] Waiting for ClusterServiceVersion "default/example-operator.v0.1.0" to reach 'Succeeded' phase
INFO[0012]   Waiting for ClusterServiceVersion "default/example-operator.v0.1.0" to appear
INFO[0021]   Found ClusterServiceVersion "default/example-operator.v0.1.0" phase: Pending
INFO[0022]   Found ClusterServiceVersion "default/example-operator.v0.1.0" phase: Installing
INFO[0028]   Found ClusterServiceVersion "default/example-operator.v0.1.0" phase: Succeeded
INFO[0028] OLM has successfully installed "example-operator.v0.1.0